src/lib/IDE/VisualStudioCode.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import re
+import time
+
+from src.lib.application import Application
+
+
+class VisualStudioCode(Application):
+    """
+    Class for managing VSCode application
+    """
+
+    def close_ide(self):
+        """
+        Closes the IDE
+        """
+        self.press_keys("ctrl", "q")
+
+    def is_open_mta_perspective(self):
+        """
+        Checks if MTA perspective is already opened in VS Code
+
+        Returns:
+            (bool): True or False
+        """
+        try:
+            self.wait_find_element(locator_type="image", locator="mta_config_active.png")
+            return True
+        except Exception as exc:
+            logging.debug(
+                "An error occured while finding \
+                MTA perspective tab ! {}".format(
+                    str(exc),
+                ),
+            )
+            if "No matches found" in str(exc):
+                return False
+            else:
+                raise Exception(exc)
+
+    def open_mta_perspective(self):
+        """
+        Opens MTA perspective in VSCode IDE
+        """
+        if self.is_open_mta_perspective():
+            logging.info("MTA perspective is already opened !")
+            return
+        else:
+            # Click on the MTA icon in left sidebar
+            time.sleep(10)
+            self.click_element(locator_type="image", locator="mta_config_inactive.png")
+
+    def run_simple_analysis(self, project, migration_target, packages=[]):
+        """
+        Runs analysis by adding the project and/or packages passed as argument
+
+        Args:
+            project (str): Full name of project to be analysed
+            packages (list): List of packages to be added to analysis
+
+        Returns:
+            None
+
+        Steps:
+            1) Click on MTA Configuration icon
+            2) Click new config(+) icon
+            3) Type project name in source
+            4) Click on config name and run analysis
+            5) Confirm analysis has started
+        """
+        # Wait for 'New Configuration' to become visible
+        time.sleep(5)
+        # Create new analysis configuration by clicking 'New Configuration' button
+        self.wait_find_element(locator_type="image", locator="create_new_config.png")
+        self.click_element(locator_type="image", locator="create_new_config.png")
+        # Wait for configuration page to load
+        time.sleep(3)
+
+        # Add project info to configuration
+        add_project_locator = self.image_locator("add_button.png")
```

Comment:
image_locator() method is missing for VSCode

Reply:
done